gh_actions_exporter/config.py

```diff
@@ -46,8 +46,14 @@ class Settings(BaseSettings):
     flavor_label: Optional[str] = 'flavor'
     default_cost: Optional[float] = 0.008
 
+    github_app_id: int
+    github_app_installation_id: int
+    github_app_private_key: SecretStr
+
```

Comment:
Focus on adding support of .env only.

When you'll add the github dependency method, we can add those vars in.
```suggestion
```